code/controllers/subsystem/shuttle.dm

```diff
@@ -127,14 +127,16 @@ SUBSYSTEM_DEF(shuttle)
 	var/threshold = CONFIG_GET(number/emergency_shuttle_autocall_threshold)
 	if(!threshold)
 		return
-
+	
 	var/alive = 0
 	for(var/I in GLOB.player_list)
 		var/mob/M = I
 		if(M.stat != DEAD)
 			++alive
 
 	var/total = GLOB.joined_player_list.len
+	if(total <= 0)
```

Comment:
If a list ever has negative length then I'd be very very curious how that came about

Reply:
this is more a force of habit for me than anything else.